app/src/main/java/com/ruslanchikbananchik/rmma/retrofit/RetrofitClient.kt

```diff
@@ -0,0 +1,19 @@
+package com.ruslanchikbananchik.rmma.retrofit
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClient {
+    private var retrofit: Retrofit? = null
+
+    fun getClient(baseUrl: String): Retrofit {
+        if (retrofit == null) {
```

Comment:
Not thread-safe implementation. See https://en.wikipedia.org/wiki/Singleton_pattern
In Kotlin you can use `by lazy { ... }` delegate. See https://kotlinlang.org/docs/delegated-properties.html#lazy-properties. It creates an instance on the first call and is thread-safe by default.